From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 9 May 2011 13:29:19 +0100
Subject: typos

Fix spelling errors and typos in strings and man pages and comments.
Change "assigment" to "assignment".
Change "pallette" to "palette".
Change "amout" to "amount".
Change "arguement" to "argument".
Change "targetted" to "targeted".
---
 CHANGES-0.8                        |    2 +-
 src/ComTerp/assignfunc.h           |    2 +-
 src/InterViews/hit.cc              |    2 +-
 src/OverlayUnidraw/setattrbyexpr.h |    2 +-
 src/man/man1/drawtool.1            |    2 +-
 src/man/man3/Control.3             |    2 +-
 src/man/man3/Viewer.3              |    2 +-
 src/man/man3/statevars.3           |    2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/CHANGES-0.8 b/CHANGES-0.8
index c0c602e..120b0f0 100644
--- a/CHANGES-0.8
+++ b/CHANGES-0.8
@@ -777,7 +777,7 @@ Interpreter Changes
  
 	* add LISP-like symbol assignment semantics to comterp.  Now
 	expressions that return a symbol can appear on the right hand
-	or left hand side of an assigment operator.  See new command
+	or left hand side of an assignment operator.  See new command
 	"symvar", and "symbol" has been renamed "symval".
 
 	* add support for command aliases to comterp, to allow
diff --git a/src/ComTerp/assignfunc.h b/src/ComTerp/assignfunc.h
index d54db4e..724f3a9 100644
--- a/src/ComTerp/assignfunc.h
+++ b/src/ComTerp/assignfunc.h
@@ -37,7 +37,7 @@ public:
     virtual boolean post_eval() { return true; }
     virtual void execute();
     virtual const char* docstring() { 
-      return "= is the assigment operator"; }
+      return "= is the assignment operator"; }
 
     CLASS_SYMID("AssignFunc");
 };
diff --git a/src/InterViews/hit.cc b/src/InterViews/hit.cc
index b9a79d9..fa6b8c4 100644
--- a/src/InterViews/hit.cc
+++ b/src/InterViews/hit.cc
@@ -379,7 +379,7 @@ void HitImpl::add_item(
 
 /*
  * The way we specify targets is the opposite of how we store them:
- * "item(depth, 0)" is the most recently targetted item at that depth.
+ * "item(depth, 0)" is the most recently targeted item at that depth.
  * So, we use "used_ - 1 - t" as the appropriate target index.
  */
 
diff --git a/src/OverlayUnidraw/setattrbyexpr.h b/src/OverlayUnidraw/setattrbyexpr.h
index 35b03b6..5bd34cf 100644
--- a/src/OverlayUnidraw/setattrbyexpr.h
+++ b/src/OverlayUnidraw/setattrbyexpr.h
@@ -37,7 +37,7 @@ class AttrDialog;
 //: command to set attributes on a component by evaluating an expression.
 // This command checks for a non-empty selection in the current editor,
 // then pops up a dialog box for entering attribute expressions to evaluate.
-// symbols on the right hand side of an assigment operator are used to
+// symbols on the right hand side of an assignment operator are used to
 // look up and return values from the property list (AttributeList) of
 // a component.  symbols on the left-hand side of an assignment operator are
 // used to create or modify an entry in the same component property list.
diff --git a/src/man/man1/drawtool.1 b/src/man/man1/drawtool.1
index 8f6f54d..fbf67b0 100644
--- a/src/man/man1/drawtool.1
+++ b/src/man/man1/drawtool.1
@@ -20,7 +20,7 @@ the following features:
 - graphical object attribute annotations
 .PP
 drawtool is derived from idraw, and implements the same set of
-pulldown commands and pallette of toolbuttons.  Read the idraw man
+pulldown commands and palette of toolbuttons.  Read the idraw man
 page for details on that part of the drawtool user interace.  This
 page lists only the extensions to idraw embodied in drawtool.
 .PP
diff --git a/src/man/man3/Control.3 b/src/man/man3/Control.3
index ca0ba08..633ac3e 100644
--- a/src/man/man3/Control.3
+++ b/src/man/man3/Control.3
@@ -99,7 +99,7 @@ Grab reads input and only passes through an event if its target
 is an active control.
 .TP
 .B "virtual void Skip()"
-Skip reads and discards input events until it reads an event targetted
+Skip reads and discards input events until it reads an event targeted
 for a control that is currently grabbing input, as indicated by
 IsGrabbing (see below).
 .TP
diff --git a/src/man/man3/Viewer.3 b/src/man/man3/Viewer.3
index 37b7ea3..13523bf 100644
--- a/src/man/man3/Viewer.3
+++ b/src/man/man3/Viewer.3
@@ -106,7 +106,7 @@ Set and get the viewer's orientation parameter, which specifies
 whether the graphical view, page, and grid will appear in portrait or
 landscape mode.  No transformation will be applied in portrait mode,
 while landscape mode will rotate the page -90 degrees and translate it
-vertically by an amout equal to the width of the page's bounding box.
+vertically by an amount equal to the width of the page's bounding box.
 .TP
 .B "virtual Selection* GetSelection()"
 Return the enclosing editor's selection object; it is equivalent to
diff --git a/src/man/man3/statevars.3 b/src/man/man3/statevars.3
index 3af2428..010a097 100644
--- a/src/man/man3/statevars.3
+++ b/src/man/man3/statevars.3
@@ -102,7 +102,7 @@ operation is performed on the component.
 .ns
 .TP
 .B "virtual void NameVar::SetName(const char*)"
-The NameVar constructor takes a string as an optional arguement, while
+The NameVar constructor takes a string as an optional argument, while
 GetName and SetName retrieve and assign the string.
 .TP
 .B "CompNameVar(Component* = nil)"
